content/en/core/releases/4.3.0.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "4.3.0 release notes"
+linkTitle: "4.3.0"
+weight:
+description: >
+relevantLinks: >
+toc_hide: true
+---
+
+## Known issues
+
+Check the repository for the [latest known issues](https://github.com/medic/cht-core/issues?q=is%3Aissue+label%3A%22Affects%3A+4.3.0%22).
+
+## Upgrade notes
+
+Continuous downwards replication (the algorithm through which offline users download docs from the server) has been completely rewritten. This change required a high number of view updates, which implies that staging this upgrade and indexing views before upgrading will be a lengthy process - depending on the size of the database. Additionally, the server might need additional storage while this process is ongoing.
+Users will receive the usual upgrade popup, and they will need to reload the app in order to resume replication.
+
+### Breaking changes
+
+None.
+
+### UI/UX changes
+
+None.
+
+## Highlights
+
+### Adding phone numbers to patients when registering via SMS
+
+The registration transition now supports storing a phone number field for new patients. This is achieved through an additional field in the SMS form, which can be configured to be parsed and validated as a correct and, optionally unique, phone number. [links to documentation]
+
+### Rewritten in downwards replication
+
+Downwards replication has been completely rewritten to improve performance, reduce complexity, remove the necessity of deleted documents tombstones and progress towards supporting eventual consistency, in order to allow for the CHT to work seamlessly with CouchDb high availability setups (multiple replicas).
+There are no UI/UX changes associated with this update, however users will need to reload the app after the upgrade was successful and they receive the notification popup, in order to resume replication using the new algorithm. The endpoints that the previous replication algorithm was using have been updated or disabled.
```

Comment:
How hard would it be to quantify the performance gain? I'd love to see a graph here!

Reply:
Ok, I'll try to find a way to get a graph. I'll try to compare telemetry sync times for a user with many docs and purged docs and see what I get.

Reply:
Added a chart.